src/docs/tutorials/proximity-steering.md

```diff
@@ -0,0 +1,203 @@
+---
+title: Setting up Proximity Steering using Cloudflare
+---
+
+### What is Proximity Steering?
+
+>Proximity steering routes visitors ... to the closest physical data center.
+
+*Source: [Proximity steering](https://developers.cloudflare.com/load-balancing/understand-basics/traffic-steering/steering-policies/proximity-steering/)*
+
+## About this Tutorial
+
+As Railway does not offer native Proximity Steering at this time, we instead need to place Cloudflare in front of our services to do this for us.
+
+This tutorial aims to provide a simple step-by-step guide on setting everything up on Cloudflare to ensure Proximity Steering works flawlessly!
+
+**Objectives**
+
+In this tutorial, you will learn how to -
+
+- Create a Health monitor.
+- Create pools for each region.
+- Set up the Proximity Load Balancer.
+
+**Prerequisites**
+
+- Have two or more identical services deployed in two or more different regions.
+
+
+    Duplicating a service can be done by right clicking and selecting **Duplicate**, opening its service settings and changing the region, then clicking **Deploy**.
+
+    The services you have deployed into the multiple regions should **only** have a Railway generated domain on them, it also helps to have the domains indicate the region.
```

Comment:
maybe - "The services should be configured with a Railway-generated domain, do not assign a custom domain.  It is also helpful to indicate the region in the domain."

Reply:
that flows much better!